.jenkins/Packaging.Jenkinsfile

```diff
@@ -78,6 +78,7 @@ pipeline {
               sh 'bash ./scripts/test-build-config -p SGX1 -b Release --build_package'
               azureUpload(storageCredentialId: 'oejenkinsciartifacts_storageaccount', filesPath: 'build/*.deb', storageType: 'blobstorage', virtualPath: 'master/${BUILD_NUMBER}/Release/SGX1/', containerName: 'oejenkins')
               azureUpload(storageCredentialId: 'oejenkinsciartifacts_storageaccount', filesPath: 'build/*.deb', storageType: 'blobstorage', virtualPath: 'master/latest/Release/SGX1/', containerName: 'oejenkins')
+              azureUpload(storageCredentialId: 'oedownload_id', filesPath: 'build/output/bin/oeedger8r', storageType: 'blobstorage', virtualPath: 'master/${BUILD_NUMBER}/oeedger8r/', containerName: 'binaries')
```

Comment:
would it be possible to some how record which revision of master the builds correspond to?